ext/exif/exif.c

```diff
@@ -3152,14 +3156,13 @@ static bool exif_process_IFD_in_MAKERNOTE(image_info_type *ImageInfo, char * val
 
 	if (value_len < 2 || maker_note->offset >= value_len - 1) {
 		/* Do not go past the value end */
-		exif_error_docref("exif_read_data#error_ifd" EXIFERR_CC, ImageInfo, E_WARNING, "IFD data too short: 0x%04X offset 0x%04X", value_len, maker_note->offset);
```

Comment:
I think that this removal causes two tests to fail (regardless of platform).

Reply:
Thanks, I've restored this in https://github.com/php/php-src/commit/2d98f0cad266c6320e1800f2c6e81847deda51c4.